sdk/storage/azure-storage-queue/azure/storage/queue/_queue_client.py

```diff
@@ -372,10 +374,10 @@ def get_queue_access_policy(self, **kwargs: Any) -> Dict[str, AccessPolicy]:
         """
         timeout = kwargs.pop('timeout', None)
         try:
-            _, identifiers = self._client.queue.get_access_policy(
+            _, identifiers = cast(List, self._client.queue.get_access_policy(
```

Comment:
This isn't exactly the correct type. It actually returns a tuple. Check out the return type of `return_headers_and_deserialized` to see full return type.